src/DIRAC/Core/scripts/dirac-agent.py

```diff
@@ -17,6 +17,9 @@
 from DIRAC.Core.Base.AgentReactor import AgentReactor
 from DIRAC.Core.Utilities.DErrno import includeExtensionErrors
 
+print("NOTE:", __file__, "is deprecated and will be removed in v7r3, for details see",
+      "https://github.com/DIRACGrid/DIRAC/wiki/DIRAC-v7r2#rename-of-scripts")
+
```

Comment:
Why isn't the `main()` from `dirac_agent.py` called here?

Reply:
This (i.e. copying the entire thing) was what we talked about during the BiLD but yeah I see no reason not to import instead 👍